### PR TITLE
Merge Jakarta namespace migration to the main branch

### PR DESCRIPTION
**Pull request by ozangunalp**
The current main e3a4a5cd576afe11ddd427649373ea52c1b38ac4 to become 3.x branch

### CODE REVIEW

File: .github/workflows/build-main-branches.yml [hunk 1]
```diff
@@ -5,8 +5,7 @@ on:
     push:
         branches:
             - main
-            - 2.x
```
Comment (cescoffier) [inline .github/workflows/build-main-branches.yml:8]: We may need to communicate that 2.x is now abandoned.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ into [CDI](http://www.cdi-spec.org/) beans,or [JAX-RS](https://github.com/eclips
 
 ## Branches
 
-* main - 3.x development stream. Uses Vert.x 4.x and Microprofile 4.x
+* main - 4.x development stream. Uses Vert.x 4.x, Microprofile 5.x and Jakarta 9
```
Comment (cescoffier) [inline README.md:22]: Isn't it 10? 
Should we mention Flow?

File: api/revapi.json [hunk 3]
```diff
@@ -27,7 +27,13 @@
     "criticality" : "highlight",
     "minSeverity" : "POTENTIALLY_BREAKING",
     "minCriticality" : "documented",
-    "differences" : [ ]
+    "differences" : [
+        {
+            "regex": true,
+            "code": ".*",
```
Comment (cescoffier) [inline api/revapi.json:33]: @jponge see how to tame revapi!

**Review by cescoffier — DISMISSED** [review]
Just a few comments.

File: .github/workflows/build-main-branches.yml (hunk 1 above)
Comment (ozangunalp) [reply .github/workflows/build-main-branches.yml:8]: I am going to make an announcement in Discussions.

File: README.md (hunk 2 above)
Comment (ozangunalp) [reply README.md:22]: I thought so at first. But 10 is in for MP 6.

Comment (ozangunalp) [reply README.md:22]: For flow I'll rebase the PR #1896 and merge it. We can mention it there.

File: api/revapi.json (hunk 3 above)
Comment (ozangunalp) [reply api/revapi.json:33]: Actually, I can clear them because we've already released a 4.2.0 version

**Review by cescoffier — APPROVED** [review]

**Pull request merged**